src/urllib3/util/ssl_.py

```diff
@@ -320,6 +328,16 @@ def create_urllib3_context(
 
     context.options |= options
 
+    if verify_flags is None:
+        verify_flags = 0
+        # In python 3.13+ ssl.create_default_context() sets VERIFY_X509_PARTIAL_CHAIN
+        # and VERIFY_X509_STRICT so we do the same
+        if sys.version_info >= (3, 13):
+            verify_flags |= VERIFY_X509_PARTIAL_CHAIN
+            verify_flags |= VERIFY_X509_STRICT
+
+    context.verify_flags |= verify_flags
```

Comment:
I can see errors like `AttributeError: 'PyOpenSSLContext' object has no attribute 'verify_flags'` in CI. urllib3 [supports](https://urllib3.readthedocs.io/en/stable/reference/contrib/pyopenssl.html) PyOpenSSL as an alternative TLS backend.

To fix the error, you can take an approach with a setter and a getter similarly to how `options` are managed. 
https://github.com/urllib3/urllib3/blob/3e8f2db735dcaced6a3b777aa1966f40c018af7c/src/urllib3/contrib/pyopenssl.py#L428-L435

It looks like PyOpenSSL [provides](https://stackoverflow.com/a/64968483) a way to set the flags, but I see no public method returning set flags.

Reply:
Do we really need to know (get) the default `verify_flags`? The `options` setter does not bother with this either and just applied the options that are set.